drivers/dht/dht.c

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright 2015 Ludwig Ortmann
+ * Copyright 2015 Christian Mehlis
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     driver_dht
+ * @{
+ *
+ * @file
+ * @brief       Device driver implementation for the DHT 11 and 22
+ *              temperature and humidity sensor
+ *
+ * @author      Ludwig Ortmann <ludwig.ortmann@fu-berlin.de>
+ * @author      Christian Mehlis <christian@m3hlis.de>
```

Comment:
@LudwigOrtmann please change my mail address to "<mehlis@inf.fu-berlin.de>" to match to the other places in RIOT